### PR TITLE
account-service should replace PROVIDER_BASE_URL with WEBHOOK_BASE_URL

### LINKED ISSUE #218: Microservices that use a webhook should replace PROVIDER_BASE_URL with WEBHOOK_BASE_URL
## Description
The gateway microservices that use webhooks previously used the environment variable `PROVIDER_BASE_URL` as the webhook target server. The `social-app-template` has adopted a new nomenclature using `WEBHOOK_BASE_URL`. All micro services should be in sync with the `social-app-template` and use the new nomenclature.

## Acceptance Criteria

All microservices that use webhooks use `WEBHOOK_BASE_URL` to provision their webhook.
- [ ] #350 
- [x] #355
- [x] content-watcher
- [ ] #356

Follow up to remove entirely: https://github.com/ProjectLibertyLabs/gateway/issues/339

### PR DESCRIPTION
**Pull request by mattheworris**
## Description
The gateway microservices that use webhooks previously used the environment variable `PROVIDER_BASE_URL` as the webhook target server. The `social-app-template` has adopted a new nomenclature using `WEBHOOK_BASE_URL`. All micro services should be in sync with the `social-app-template` and use the new nomenclature.

## Acceptance Criteria

- All microservices that use webhooks use `WEBHOOK_BASE_URL` to provision their webhook.
  - [x] account

Follow up to remove entirely: https://github.com/ProjectLibertyLabs/gateway/issues/339

### CODE REVIEW

File: docker-compose-k6.account.yaml [hunk 1]
```diff
@@ -32,8 +31,7 @@ services:
       - redis
       - ipfs
     environment:
-      PROVIDER_BASE_URL: 'http://account-service-webhook:3001/webhooks/account-service'
-      WEBHOOK_BASE_URL: 'http://account-service-webhook:3001/webhooks'
```
Comment (mattheworris) [inline docker-compose-k6.account.yaml:36]: Note: I didn't see where `WEBHOOK_BASE_URL` was used in the code, so I think it was redundant.

File: services/account/README.md [hunk 2]
```diff
@@ -180,17 +180,6 @@ In order to run this project you need:
 - [Node.js](https://nodejs.org)
 - [Docker](https://docs.docker.com/get-docker/)
 
-### Setup
```
Comment (mattheworris) [inline services/account/README.md:183]: Not Applicable in the mono-repo.

File: services/account/README.md [hunk 3]
```diff
@@ -209,6 +198,7 @@ Install NPM Dependencies:
 
 ```sh
   npm install
+  npm install -g @nestjs/cli
```
Comment (mattheworris) [inline services/account/README.md:201]: Not sure about this one, but my system could not find the `nest` command.

File: services/account/apps/api/src/services/api.service.ts [hunk 4]
```diff
@@ -15,13 +15,4 @@ export class ApiService implements OnApplicationShutdown {
       this.logger.error(`Error during cleanup on shutdown: ${e}`);
     }
   }
-
```
Comment (mattheworris) [inline services/account/apps/api/src/services/api.service.ts:18]: Removed old code.

File: services/account/scripts/docker-clean-up.sh [hunk 5]
```diff
@@ -2,12 +2,13 @@
...
 # Stop and remove containers, networks
 echo "Stopping and removing containers, networks..."
-docker compose down
+docker compose down frequency redis account-service-api account-service-worker
 
 # Remove specified volumes
 echo "Removing specified volumes..."
-docker volume rm account-service_redis_data
-docker volume rm account-service_chainstorage
+docker volume rm gateway_redis_data
```
Comment (mattheworris) [inline services/account/scripts/docker-clean-up.sh:9]: docker-compose.yaml file moved to root, so targets needed to be renamed to match.

File: services/account/README.md (hunk 3 above)
Comment (mattheworris) [reply services/account/README.md:201]: `account-service` uses `nest` in a bunch of the `package.json` scripts. I probably lost my `nest` when switching from asdf to nvm. We could also add `@nestjs/cli` to the dev-dependencies.

Comment (wilwade) [reply services/account/README.md:201]: > add @nestjs/cli to the dev-dependencies

At least on that commit it shows up as a full dependency: https://github.com/ProjectLibertyLabs/gateway/blob/859ab33ab96e359075085e3a4908d3ff8e8446a2/services/account/package.json#L56

File: services/account/libs/common/src/services/provider-webhook.service.ts
Comment (JoeCap08055) [inline services/account/libs/common/src/services/provider-webhook.service.ts:1]: nit: rename this file?

**Pull request merged**